converter/src/test/java/gov/cms/qpp/acceptance/cpc/CpcPlusAcceptanceTest.java

```diff
@@ -60,18 +72,64 @@ void testCpcPlusFileFailures() throws IOException {
 		List<Path> successesThatShouldBeErrors = getXml(FAILURE)
 			.filter(entry -> {
 				Converter converter = new Converter(new PathQrdaSource(entry));
-
+				//TODO remove this once all error scenarios are validated
+				if (fixtureValues.get(entry.toFile().getName()) == null) {
+					return false;
+				}
 				try {
 					converter.transform();
 					return true;
 				} catch (TransformException expected) {
+					//runnning conversions on individual files
+					List<Detail> details = expected.getDetails().getErrors().get(0).getDetails();
```

Comment:
Is there any way to refactor how the `TransformException` works with retrieving the details?  This is a long stream of function calls that's not real easy to understand.

Reply:
Not sure we have a usage for this outside of this test class. Will give it a look though.

Reply:
sounds good.  if it's not being used anywhere else I'm ok with it.